doc/man1/openssl-fipsinstall.pod.in

```diff
@@ -174,6 +175,18 @@ target machine. Once the self tests have run on the target machine the user
 could possibly then add the 2 fields into the configuration using some other
 mechanism.
 
+This is the default.
+
+=item B<-self_test_oninstall>
+
+The converse of B<-self_test_oninstall>.  The two fields related to the
+"test status indicator" and "MAC status indicator" are written to the
+output configuration file.
+
+This option allows OpenSSL 3.0 FIPS providers to be installed so that
+the self tests run only at installation time.  It results in an error
+if applied to the OpenSSL 3.1 FIPS provider series.
```

Comment:
No it doesn't - and it shouldn't. This is a users choice to make.

Reply:
The underlying code no longer support this mode of operation.  The self tests always run.

Reply:
They should not always run. We should have the same options we have under the existing module. You can change the default behaviour if you wish - but the same handling should remain.